code/modules/client/preference_setup/preference_setup.dm

```diff
@@ -34,7 +34,7 @@
 	category_item_type = /datum/category_item/player_setup_item/skills
 
 /datum/category_group/player_setup_category/occupation_preferences
-	name = "Occupation"
+	name = "Occupation / Faction"
```

Comment:
"Occupation and Faction" might be better here? i don't know what context its used in.